### PR TITLE
add classes for css and javascript selection

### PR DESCRIPTION
**Pull request by taoufiqaitali**
help to customise payment step page

| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | help to customise payment step on checkout page.
| Type?         | improvement 
| Category?     | FO
| BC breaks?    | no
| Deprecations? |no
| How to test?  |  checkout payment step

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/10398)

### CODE REVIEW

File: themes/classic/templates/checkout/_partials/steps/payment.tpl [hunk 1]
```diff
@@ -10,6 +10,7 @@
   <div class="payment-options {if $is_free}hidden-xs-up{/if}">
     {foreach from=$payment_options item="module_options"}
       {foreach from=$module_options item="option"}
+        <div class="eachpaymentoption {$option.module_name} eachpaymentoption-{$option.id} ">
```
Comment (MathiasReker) [inline themes/classic/templates/checkout/_partials/steps/payment.tpl:13]: Please run cs-fixer :-)

Comment (taoufiqaitali) [reply themes/classic/templates/checkout/_partials/steps/payment.tpl:13]: i did in last commit

Comment (MathiasReker) [reply themes/classic/templates/checkout/_partials/steps/payment.tpl:13]: 👍

**Comment by MathiasReker** [comment]
Branch should be 'develop' only. Please remove 1.7.4.x

**Comment by MathiasReker** [comment]
I see that there are no selected payment method by default. I think it would be the right behavior to have the first payment method selected by default. What do you think @taoufiqaitali ? (maybe this should be done in an other PR)

**Comment by taoufiqaitali** [comment]
> I see that there are no selected payment method by default. I think it would be the right behavior to have the first payment method selected by default. What do you think @taoufiqaitali ? (maybe this should be done in an other PR)

yeah this should be added, more friendly UX,, yes in new PR

**Comment by MathiasReker** [comment]
@taoufiqaitali would you be willing to give it a shot? :-)

**Comment by Progi1984** [comment]
@NeOMakinG May be it could be backported on `develop` branch ?

**Comment by NeOMakinG** [comment]
@Progi1984 I doubt that my other PR adds these one, that would be cool yes !

**Comment by Progi1984** [comment]
@NeOMakinG Could you backport it and close this one ?

**Comment by PierreRambaud** [comment]
Hi everyone,

Since we had no news, I decide to close this pull request.
Feel free to reopen, backport or whatever you want with it if you think it's still relevant.

Kind regards

**Pull request closed without merging**